src/validation.cpp

```diff
@@ -192,12 +192,11 @@ bool CheckFinalTx(const CTransaction &tx, int flags)
 {
     AssertLockHeld(cs_main);
 
-    // By convention a negative value for flags indicates that the
-    // current network-enforced consensus rules should be used. In
-    // a future soft-fork scenario that would mean checking which
-    // rules would be enforced for the next block and setting the
-    // appropriate flags. At the present time no soft-forks are
-    // scheduled, so no flags are set.
+    // By convention a negative value for flags indicates that no
+    // soft-forked consensus rules should be used. Setting appropriate
+    // flags select which rules would be enforced for the next block.
```

Comment:
Here it seems should be either "Setting-appropriate flags select" or "Setting appropriate flags selects". 

I think it's the latter but it's not clear from the sentence.

Suggest: s/would/will/